frontend/src/components/pages/shadowlinks/details/config/configuration-shadowing.tsx

```diff
@@ -166,16 +166,38 @@ const ACLFilterSection = ({ filters }: { filters: ACLFilter[] }) => {
   );
 };
 
+// Component to display Schema Registry sync status
+const SchemaRegistrySection = ({ isEnabled }: { isEnabled: boolean }) => (
+  <Card size="full" testId="schema-registry-card">
+    <CardHeader>
+      <Heading level={3}>Schema Registry</Heading>
+    </CardHeader>
+    <CardContent className="flex flex-row justify-between">
+      <Text className="mt-2 text-muted-foreground text-sm">
+        The _schemas topic is being replicated byte-for-byte from the source cluster.
+      </Text>
```

Comment:
```suggestion
      <Text className="mt-2 text-muted-foreground text-sm">
        The _schemas topic is replicated byte-for-byte from the source cluster.
      </Text>
```

IIUC, this is the configuration card to toggle the schema registry sync, right?. If that so, `Being replicated` can imply an active background process, even when the feature is disabled. wdyt?

Reply:
yes, looks good!